op-acceptance-tests/tests/interop/interop_test_helpers.go

```diff
@@ -90,7 +93,7 @@ func DeployEventLogger(ctx context.Context, wallet system.WalletV2, logger log.L
 		return common.Address{}, err
 	}
 	eventLoggerAddress := res.ContractAddress
-	logger.Info("Deployed EventLogger", "address", eventLoggerAddress)
+	logger.Info("Deployed EventLogger", "chainID", deployTx.ChainID.Value().String(), "address", eventLoggerAddress)
```

Comment:
Nit: `.String()` gets called by the logging library already, and the value doesn't mutate, so you don't have to call it here.

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/15049/commits/486c6039d09276b603613142a84197b75e05188a